app/js/components/subAddress/addressForm.jsx

```diff
@@ -325,7 +325,7 @@ export default class AddressForm extends React.Component {
             </Modal>
           :
           <div>
-            {this.state.secondryAction === 'new' ?
+            {this.props.address.secondryAction === 'new' ?
```

Comment:
Is this the only change required to Add plus icon for addresses?

Reply:
Yes it is.

Reply:
So do you now see how easier it is to review the actual change in comparison to when you had all the other formatting changes?

Reply:
Yes it is, sorry about that.

Reply:
And by the way, formatting and cleanup is a very good thing. It is just that it should not mixed with other changes. 😊